decidim/decidim
# Tool tip text displayed with tags on "Visit URL" link

#### :tophat: What? Why?
On "Global Moderation" section from admin dashboard, the tool tip for "Visit URL" text contains tags.


#### :pushpin: Related Issues
Bug opened on metadecidim: https://meta.decidim.org/processes/bug-report/f/210/proposals/15837
![Screenshot from 2020-12-16 13-45-48](https://user-images.githubusercontent.com/66411127/102348561-c199bb80-3faa-11eb-86a8-af9cd17bdf24.png)


#### Testing
1. Login as admin and go to Admin dashboard
2. Create a meeting and report it from the interface
3. Go to global moderation
4. Check the tool tip.

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
